be/src/olap/schema_change.cpp

```diff
@@ -1383,6 +1463,15 @@ OLAPStatus SchemaChangeHandler::_do_process_alter_tablet_v2(const TAlterTabletRe
         sc_params.new_tablet = new_tablet;
         sc_params.ref_rowset_readers = rs_readers;
         sc_params.delete_handler = delete_handler;
+        if (request.__isset.materialized_view_params) {
+            for (auto item : request.materialized_view_params) {
+                AlterMaterializedViewParam mvParams;
```

Comment:
Encapsulate the logic here. There will be many new rules for converting expressions into strings